src/taipy/core/_entity/_migrate.py

```diff
@@ -344,7 +352,7 @@ def __write_entities_to_mongo(
         )
 
 
-def _load_all_entities_from_sql(db_file: str = "test.db") -> Tuple[Dict, Dict]:
+def _load_all_entities_from_sql(db_file: str) -> Tuple[Dict, Dict]:
```

Comment:
We should have as a default value the default value of the configuration.
What do you think?

Reply:
The `db_file` should always be provided by the user and should be processed at the CLI level.